Cython/Includes/cpython/datetime.pxd

```diff
@@ -237,12 +235,8 @@ cdef extern from "datetime.h":
     int PyDateTime_DELTA_GET_MICROSECONDS(object o)
 
     # Constructors
-    object PyTimeZone_FromOffset(object offset)
-    object PyTimeZone_FromOffsetAndName(object offset, object name)
-
-    # Constructors for the DB API
-    object PyDateTime_FromTimeStamp(object args)
-    object PyDate_FromTimeStamp(object args)
```

Comment:
Please keep these so that they can be used by developers who target Python 3.7 and later.